src/checkers/inference/InferenceVisitor.java

```diff
@@ -848,4 +860,109 @@ protected void checkConstructorResult(
         }
     }
 
+    /**
+     * This method creates a mapping from type-use locations a lists of qualifiers.
```

Comment:
```suggestion
     * This method creates a mapping from type-use locations to a set of qualifiers which cannot be applied to that location.
```

Please reformat the line break...